db/migrate/20191115200413_update_nt_info_to_berkeley_db.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateNtInfoToBerkeleyDb < ActiveRecord::Migration[5.1]
+  def change
+    AlignmentConfig.where(name: "2019-09-17").update_all(
+      s3_nt_info_db_path: "s3://idseq-database/alignment_data/2019-09-17/nt_info.db",
```

Comment:
Conceptually, isn't that a new alignment config? I worry that by updating it (rather than creating a new one), we could lose documentation of how a sample was run. Although the reference files used are also reflected in the DAG JSON that gets produced, so maybe we just need to be aware that the DAG is the source of truth for how a sample was run, not the alignment config the pipeline_run is pointed to.

Reply:
These configs should both be identical key value stores that are stored in a different format so the internal data should be the same. I know it has a different path so there may be some confusion, but I just wanted to make this clear.

We want to eliminate the sqlite3 dependency at runtime which means we no longer want to keep supporting this as per @boris-dimitrov 's cleanup initiatives. I also agree this is a good thing. The level of complexity we have introduced to do something conceptually simple is quite high. This will mean running with these older configs will break after this switch happens which is why I want to do this migration.

Also the dag JSON still will reference the file extension used so no information is lost.